word_movers_distance/Calculate_WMD_cos_scores.ipynb

```diff
@@ -0,0 +1,3817 @@
+    "\n",
+    "  ss = [article.split()]\n",
+    "  bb = build_phrases(ss)\n",
+    "  pb = [bb[i] for i in ss][0]\n",
+    "\n",
+    "  dem_score = calc_wmd(pb,(dem_core))\n",
+    "  rela_score = calc_wmd(pb,(rela_core))\n",
+    "  cult_score = calc_wmd(pb,(cult_core))\n",
+    "  return (dem_score,rela_score,cult_score)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "dkv8bM_c6XIo",
+    "outputId": "f3b2f33d-3521-4192-c6e6-43277d8800bd"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/l0/d6tznk515rlb1sw7x2cnp5pc0000gn/T/ipykernel_12900/728712704.py:2: DeprecationWarning: Call to deprecated `wmdistance` (Method will be removed in 4.0.0, use self.wv.wmdistance() instead).\n",
+      "  return w2v.wmdistance(a,b)\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "(4.198177976094678, 4.013062402306361, 3.9515340515701287)"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_sims_with_perspective(pos_cult.loc[0,'processed'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "eO7TR0lf9V0S",
+    "outputId": "d3f1a253-f842-4906-9cef-21e890e32f98"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:2: DeprecationWarning: Call to deprecated `wmdistance` (Method will be removed in 4.0.0, use self.wv.wmdistance() instead).\n",
+      "  \n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "(5.511709636011158, 5.235622040175475, 5.398345213885615)"
+      ]
+     },
+     "execution_count": 66,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_sims_with_perspective(pos_rela.loc[0,'processed'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "id": "1Ziz5EEcjtNm"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "def get_sim_perspective(df):\n",
+    "\n",
+    "    \"\"\"\n",
+    "    returns modified df with extra columns dem_score,rela_score,cult_score\n",
+    "    denoting the wmd score calculated for each article.\n",
+    "    \"\"\"\n",
+    "    tqdm.pandas()\n",
+    "\n",
+    "    results=df['processed'].apply(lambda x:get_sims_with_perspective(x))\n",
+    "    dem_scores = [i[0] for i in results]\n",
+    "    rela_scores = [i[1] for i in results]\n",
+    "    cult_scores = [i[2] for i in results]\n",
+    "    df['dem_scores'] = dem_scores\n",
+    "    df['rela_scores'] = rela_scores\n",
+    "    df['cult_scores']= cult_scores\n",
```

Comment:
There are too many columns in the output data, so let's clean that up by overwriting `dem_wmd`, etc, while also giving the main wmd result a clearer name:
```suggestion
    "    df['dem_wmd'] = dem_scores\n",
    "    df['rela_wmd'] = rela_scores\n",
    "    df['cult_wmd']= cult_scores\n",
```

Reply:
Has been addressed with this cell:
```python3
pos_cult=pos_cult.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})
neg_cult=neg_cult.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})

pos_demog=pos_demog.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})
neg_demog=neg_demog.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})

pos_rela=pos_rela.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})
neg_rela=neg_rela.rename(columns={'dem_scores':'dem_wmd', 'rela_scores':'rela_wmd', 'cult_scores':'cult_wmd'})
```